packages/material-ui/src/CircularProgress/CircularProgress.js

```diff
@@ -162,6 +180,19 @@ CircularProgress.propTypes = {
    * The color of the component. It supports those theme colors that make sense for this component.
    */
   color: PropTypes.oneOf(['primary', 'secondary', 'inherit']),
+  /**
+   * If `true`, the shrink animation is applied. This only works if variant is `indeterminate`
+   */
+  shrinkAnimation: chainPropTypes(PropTypes.bool, props => {
```

Comment:
This should be named `disableShrinkAnimation` according to https://material-ui.com/guides/api/#property-naming

Reply:
Thanks for the feedback! This makes a lot more sense, changed shrinkAnimation to disableShrinkAnimation